.azure/pipelines/ci-official.yml

```diff
@@ -7,24 +7,77 @@ variables:
   BuildConfiguration: Release
 
 jobs:
+- job: Docker_arm
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+    
+  - script: |
+      dotnet build --configuration $(BuildConfiguration)
+      dotnet publish --runtime linux-arm --output publish/arm src/BaGet --configuration $(BuildConfiguration)
+    displayName: Build & Publish BaGet-ARM
+
+  - task: Docker@1
+    displayName: Build docker image-arm
+    inputs:
+      command: build
+      dockerFile: arm.Dockerfile
+      imageName: loicsharma/baget:stretch-slim-arm32v7-$(Build.SourceVersion)
+  - task: Docker@1
+    displayName: Tag latest docker image-arm
+    condition: eq(variables['build.sourcebranch'], 'refs/heads/dev')
+    inputs:
+      command: tag
+      imageName: loicsharma/baget:stretch-slim-arm32v7-$(Build.SourceVersion)
+      arguments: loicsharma/baget:stretch-slim-arm32v7
+
+  - task: Docker@1
+    displayName: Push docker image
+    condition: eq(variables['build.sourcebranch'], 'refs/heads/dev')
+    inputs:
+      command: push
+      containerRegistryType: 'Container Registry'
+      dockerRegistryEndpoint: 'Docker Hub'
+      imageName: loicsharma/baget
+
+
 - job: Docker
   pool:
     vmImage: ubuntu-16.04
   steps:
+
   - task: Docker@1
     displayName: Build docker image
     inputs:
       command: build
       dockerFile: Dockerfile
       imageName: loicsharma/baget:$(Build.SourceVersion)
 
+  - script: dotnet publish --configuration Release --runtime linux-arm --output publish/arm $(BuildConfiguration)
```

Comment:
Why do you modify this Azure DevOps job too?

Reply:
I lost those lines, sorry.